### PR TITLE
PostCSS Font Format Keywords

### PR DESCRIPTION
**Pull request by Antonio-Laguna**
Initial work for #116

### CODE REVIEW

File: plugins/postcss-font-format-keywords/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+@font-face {
+  src: url(file.woff2) format(woff2);
+}
+
+/* becomes */
+
+@font-face {
+  src: url(file.woff2) format('woff2');
+}
+```
```
Comment (romainmenke) [inline plugins/postcss-font-format-keywords/README.md:70]: Is there a technical reason to have this option?
If there isn't, I would remove it.

File: plugins/postcss-font-format-keywords/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "description": "Use unquoted format on @font-face CSS definitions.",
+  "author": "Jonathan Neal <jonathantneal@hotmail.com>",
+  "license": "CC0-1.0",
+  "homepage": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-font-format-keywords#readme",
+  "engines": {
+    "node": "^12 || ^14 || >=16"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
```
Comment (romainmenke) [inline plugins/postcss-font-format-keywords/package.json:13]: Can we add the `exports` fields?
I haven't added it yet to the base plugin but I try to add it consistently now.

File: plugins/postcss-font-format-keywords/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		AtRule: {
+			'font-face'(atRule) {
+				if (atRule.name !== 'font-face') {
+					// Case sensitive
+					return;
+				}
+
+				atRule.walkDecls('src', decl => {
+					if (!decl.value.includes('format(')) {
+						// No point on doing parsing if no format is specified
```
Comment (romainmenke) [inline plugins/postcss-font-format-keywords/src/index.ts:32]:
```suggestion
						// No point in doing parsing if no format is specified
```

File: plugins/postcss-font-format-keywords/stryker.conf.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  "commandRunner": {
+    "command": "npm run test:tape"
+  },
+  "thresholds": {
+    "high": 100,
+    "low": 100,
+    "break": 100
+  },
+  "inPlace": true
+}
```
Comment (romainmenke) [inline plugins/postcss-font-format-keywords/stryker.conf.json:19]: This file can be removed for now.
There is no complex logic that needs unit tests.

**Review by romainmenke — COMMENTED** [review]
Some small questions and notes :)

Looks great!

File: plugins/postcss-font-format-keywords/README.md (hunk 1 above)
Comment (Antonio-Laguna) [reply plugins/postcss-font-format-keywords/README.md:70]: Other than, it was already there, there's no other reason

File: plugins/postcss-font-format-keywords/package.json (hunk 2 above)
Comment (Antonio-Laguna) [reply plugins/postcss-font-format-keywords/package.json:13]: Good catch, will do

**Review by romainmenke — APPROVED** [review]
LGTM!

**Pull request merged**